<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="detalle_ctro_entregaVentasV2-20241107174314" tests="1" file=".py" time="71.585" timestamp="2024-11-07T17:44:25" failures="1" errors="0" skipped="0">
	<testcase classname="detalle_ctro_entregaVentasV2" name="test_detail_deriverySales" time="71.585" timestamp="2024-11-07T17:44:25" file="CtaCtoDtaEntVent2.py" line="29">
		<failure type="AssertionError" message=""><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\CtaCtoDtaEntVent2.py", line 169, in test_detail_deriverySales
    validate_text_visible(self.driver, element5, text_expected)
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\Elements.py", line 1186, in validate_text_visible
    assert text_obtained == text_expected
AssertionError
]]></failure>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 1 veces con éxito!
El texto encontrado es  Estado De Mis Contratos
El texto es visible para el usuario
El texto fue validado correctamente: Contrato 124536
El texto es visible para el usuario
El texto fue validado correctamente: De Trigo
El texto es visible para el usuario: 'De 439.853,00 Kg Pactados'
El texto es visible para el usuario
El texto fue validado correctamente: Mis Entregas
El texto es visible para el usuario
El texto fue validado correctamente: Producto
El texto es visible para el usuario
]]></system-out>
	</testcase>
</testsuite>
